modelci/types/bo/model_objects.py

```diff
@@ -43,8 +43,8 @@ class Engine(Enum):
     ONNX = 3
     TRT = 4
     TVM = 5
-    PYTORCH = 6
-    CUSTOMIZED = 7
+    CUSTOMIZED = 6
```

Comment:
Why you switch these two type?

Reply:
Because the order here should be same with here, so the web page will display the right engine name
https://github.com/cap-ntu/ML-Model-CI/blob/master/modelci/types/models/common.py#L87-L95